packages/sql-openapi/test/multiple-fk.test.js

```diff
@@ -0,0 +1,196 @@
+'use strict'
+
+const { clear, connInfo, isPg, isSQLite } = require('./helper')
+const { test } = require('tap')
+const fastify = require('fastify')
+const sqlMapper = require('@platformatic/sql-mapper')
+const sqlOpenAPI = require('..')
+
+test('multiple foreign keys pointing the same table', { skip: isSQLite }, async ({ equal, same, teardown }) => {
+  async function onDatabaseLoad (db, sql) {
+    await clear(db, sql)
+
+    if (isPg) {
+      await db.query(sql`CREATE TABLE IF NOT EXISTS test1 (
+        id VARCHAR(42) PRIMARY KEY
+      );`)
+
+      await db.query(sql`CREATE TABLE IF NOT EXISTS test2 (
+        id VARCHAR(42) NOT NULL,
+        fk_id VARCHAR(42) NOT NULL,
+        custom_id VARCHAR(42) NOT NULL,
+        PRIMARY KEY (id),
+        FOREIGN KEY (custom_id) REFERENCES test1 (id),
+        FOREIGN KEY (fk_id) REFERENCES test1 (id)
+      );`)
+    } else {
+      await db.query(sql`CREATE TABLE IF NOT EXISTS test1 (
+        id VARCHAR(42) PRIMARY KEY
+      );`)
+
+      await db.query(sql`CREATE TABLE IF NOT EXISTS test2 (
+        id varchar(42) NOT NULL,
+        fk_id varchar(42) NOT NULL,
+        custom_id varchar(42) NOT NULL,
+        PRIMARY KEY (id),
+        CONSTRAINT test2_custom_id_foreign_idx FOREIGN KEY (custom_id) REFERENCES test1 (id),
+        CONSTRAINT test2_fk_id_foreign_idx FOREIGN KEY (fk_id) REFERENCES test1 (id)
+      );`)
+    }
```

Comment:
Instead of creating new table names, please reuse the ones that are already cleared up in `test/helper.js`

Reply:
I used test1/test2 tables that are already existing on the [helper](https://github.com/platformatic/platformatic/blob/main/packages/sql-openapi/test/helper.js#L103)

Reply:
My bad: I used by mistake schema names that already existed (instead of table names).
Fixed on [1d364db](https://github.com/platformatic/platformatic/pull/548/commits/1d364db8b732c3a2e4969309d3b230f30dc64529)